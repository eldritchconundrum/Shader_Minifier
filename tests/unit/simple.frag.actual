/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef SIMPLE_FRAG_ACTUAL_
# define SIMPLE_FRAG_ACTUAL_

const char *simple_frag =
 "void main()"
 "{"
   "gl_FragColor=vec4(.2,.4,.6,0.);"
 "}";

#endif // SIMPLE_FRAG_ACTUAL_
