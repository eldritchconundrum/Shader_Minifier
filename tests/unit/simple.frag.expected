/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef SIMPLE_FRAG_EXPECTED_
# define SIMPLE_FRAG_EXPECTED_

const char *simple_frag =
 "void main()"
 "{"
   "gl_FragColor=vec4(.2,.4,.6,0.);"
 "}";

#endif // SIMPLE_FRAG_EXPECTED_
